libMBIN/Source/Models/Structs/GcSolarSystemData.cs

```diff
@@ -32,30 +31,42 @@ public class GcSolarSystemData : NMSTemplate
 
         /* 0x03B0 */ public Vector2f FreighterTimer;
         /* 0x03B8 */ public Vector2f PirateTimer;
-        /* 0x03C0 */ public Vector2f FlybyTimer;
+        /* 0x03C0 */  public Vector2f FlybyTimer;
         [NMS(Size = 8, Ignore = true)]
-        /* 0x03C8 */ public byte[] Padding3C8;
-        /* 0x0390 */ public GcSpaceStationSpawnData SpaceStationSpawn;
+        /* 0x03D0 */ public byte[] Padding3C8;
+        /* 0x0468 */ public GcSpaceStationSpawnData SpaceStationSpawn;
         /* 0x0510 */ public GcSolarSystemTraderSpawnData TraderSpawnOnOutposts;
         /* 0x0524 */ public GcSolarSystemTraderSpawnData TraderSpawnInStations;
-        /* 0x0538 */ public List<GcSolarSystemLocator> Locators;
+        /* 0x0538 */ public List<GcSolarSystemLocator> Locator;
         /* 0x0548 */ public List<NMSTemplate> AsteroidGenerators;
         /* 0x0558 */ public int AsteroidLevel;
+
         public string[] AsteroidLevelValues()
         {
             return new[] { "NoRares", "SomeRares", "LotsOfRares" };
         }
-
+        
+        
         [NMS(Size = 4, Ignore = true)]
         /* 0x055C */ public byte[] Padding51C;
-        /* 0x0560 */ public GcPlanetColourData Colours;
+
+        /* 0x0560 */ public GcPlanetColourData PlanetColours;
+        
         /* 0x1750 */ public GcLightProperties Light;
```

Comment:
These offsets should have changed also since the GcPlanetColourData struct has increased in size

Reply:
They're supposed to be at 0x1C20 and 0x1C50